### ISSUE TITLE
test accuracy is only in the forties

### ISSUE THREAD

**Issue by LinXianL** [XJTU-Graph-Intelligence-Lab/Multi-Track-Message-Passing#4]
Hello. I used your code and parameters, but the test accuracy is only in the forties. May I ask what the reason is?

**Comment by XJTU-Graph-Intelligence-Lab** [comment]
Which experiment are you referring to?  Has the training process reached convergence?

**Comment by LinXianL** [comment]
I ran the Cora dataset following all the instructions you provided and without making any modifications to the code. However, the result is only in the forties.

**Comment by LinXianL** [comment]
This is an example: FrozenTrial(number=0, state=TrialState.COMPLETE, values=[0.40700000000000003], datetime_start=datetime.datetime(2025, 5, 6, 3, 9, 12, 865161), datetime_complete=datetime.datetime(2025, 5, 6, 3, 16, 28, 366140), params={'lr': 0.0015120095955493637, 'tblr': 0.001308153854135193, 'tbwd': 7.46963166452136e-05, 'tpwd': 9.516596553788267e-05, 'lw': 0.42803621204275966, 'layer_num': 64, 'dropout': 0.24838422015789424}, user_attrs={}, system_attrs={}, intermediate_values={}, distributions={'lr': FloatDistribution(high=0.05, log=True, low=0.001, step=None), 'tblr': FloatDistribution(high=0.05, log=True, low=0.001, step=None), 'tbwd': FloatDistribution(high=0.0001, log=False, low=1e-06, step=None), 'tpwd': FloatDistribution(high=0.0001, log=False, low=1e-06, step=None), 'lw': FloatDistribution(high=0.7, log=False, low=0.01, step=None), 'layer_num': CategoricalDistribution(choices=(64, 96, 132, 164, 196, 256)), 'dropout': FloatDistribution(high=0.8, log=False, low=0.1, step=None)}, trial_id=0, value=None)

**Comment by liyu199809** [comment]
Thank you for bringing this to my attention! 

After re-running the parameter tuning experiments, I identified the issue stems from an incorrect setting of **args.a**, which represents the residual connection coefficient in the track mechanism. For optimal performance, **please use the default value of 0.9 for this parameter.**

Additionally, I recommend including args.a in your Optuna hyperparameter search. A suggested approach is to first explore a broad range of values to observe trends, then conduct multiple fine-tuning rounds within a narrower range to refine the optimal setting. This two-step strategy helps balance exploration and exploitation in hyperparameter optimization.


I have fixed this error in the latest code. Please let me know if you need further clarification or assistance with the implementation – I’m happy to help ensure your replication proceeds smoothly!
